### ISSUE TITLE
Meaningful link text for TOC tools

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/tr-design#136]
This was [filed against WCAG 2.1](https://github.com/w3c/wcag21/issues/158) but it comes from the W3C styles. Quoting the original issue:

    <a id="toc-jump" href="#toc"><span aria-hidden="true">↑</span> <span>Jump to Table of Contents</span></a>

    and

    <a id="toc-toggle" href="#toc"><span aria-hidden="true">→</span> <span>Pop Out Sidebar</span></a>

When I look at the code...my brain thinks..."I see link text...it should be fine."  But when I tried it in VoiceOver, NVDA and JAWS...alas....the link text is not good in any of these 3 screen readers. 

[An issue comment provides further details of the user experience](https://github.com/w3c/wcag21/issues/158#issuecomment-329956552).

**Comment by jnurthen** [comment]
I think I have previously looked at this and it needs to be fixed in fixup.js
If no one else gets to this before me I’ll look at it.

**Comment by plehegar** [comment]
would adding a title attribute on the anchor element fix this?

**Comment by thierryk** [comment]
As @jnurthen suggested, this seems related to JS as the markup looks fine.
In any case, I think `aria-label` to convey/fake "link text" to screen-readers is a better choice than `title`.

**Comment by TimvdLippe** [comment]
While running `axe` on our documents, I ran into the same issue. I wrote a fix for this issue (tested on our documents): #367
